### PR TITLE
Lint API

### PR DESCRIPTION
**Pull request by sdevalapurkar**
# Overview

This PR includes the following proposed change(s):

- Lint the API
- Down to just 3 warnings and 0 errors

## Type of change

- [x] Refactoring / Documentation

## How Has This Been Tested?

- Local

### CODE REVIEW

File: api/src/paths/activity/{activityId}.ts [hunk 1]
```diff
@@ -63,7 +63,7 @@ GET.apiDoc = {
  * @return {RequestHandler}
  */
 function getActivity(): RequestHandler {
-  return async (req, res, next) => {
```
Comment (NickPhura) [inline api/src/paths/activity/{activityId}.ts]: Hmmmm, is this the suggested course of action by the linter?

I feel like `res` is better in this case, as `_` is really just a different name for an unused parameter.  And this is more likely to result in someone actually using `_` to do stuff: `_.someAction()` type thing.

Comment (sdevalapurkar) [reply api/src/paths/activity/{activityId}.ts]: nope, just used it cause it was unused
but I agree with you, will put it back :)

**Comment by dogcomplex** [comment]
Scanned through and looking good 👍

**Review by dogcomplex — APPROVED** [review]
👍

**Pull request merged**